instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -107,9 +108,7 @@ def process_request(self, request):
 
         token = attach(extract(request_meta, getter=wsgi_getter))
 
-        tracer = get_tracer(__name__, __version__)
-
-        span = tracer.start_span(
+        span = self._tracer.start_span(
```

Comment:
`None` check?

Reply:
Ideally this wouldn't None because it is set [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/402/files#diff-c93e4cdcf380e4064033aacbb347553945bb8aa80866cdf24355f867d0917a90R114) when `.instrument()` is called but let me check if there is chance of user misconfiguring and subsequently running into error.